MANIFEST.in

```diff
@@ -16,6 +16,7 @@ recursive-include lib/ansible/module_utils/csharp *.cs
 recursive-include lib/ansible/module_utils/powershell *.psm1
 recursive-include lib/ansible/modules/windows *.ps1
 recursive-include lib/ansible/galaxy/data *.yml *.j2 README.md ansible.cfg inventory .git_keep
+recursive-include lib/ansible/collections *.py
```

Comment:
This shouldn't be needed. I believe the previous failure was only due to the lack of `__init__.py` in the directory. Python files are included by default.